networking/nsx-firewall-patterns/README.md

```diff
@@ -0,0 +1,405 @@
+          auth_pass absc123456789
+      }
+      virtual_ipaddress {
+          192.168.0.100
+      }
+   ```
+
+   - _Interface - South or North interface of your NVA/Firewall_
+   - _Auth_pass - Authentication password_
+   - _virtual_ipaddress - Virtual IP address for your NVA/Firewall. This will be shared between the two NVAs. You can pick any un-used IP in the respective transit segment._
+   - _Priority - The value decided the active node. The value should be smaller than the one assigned to active NVA, for standby NVA._
+
+
+   _For more information, please refer to the following link:_
+   [Configuration file for keepalived](https://manpages.ubuntu.com/manpages/xenial/man5/keepalived.conf.5.html)
+
+   3. Stop VRRP
+   ```
+   # sudo systemctl stop keepalived
+   ```
+   4. Start VRRP
+   ```
+   # sudo systemctl start keepalived
+   ```
+   5. Verify
+   ```
+   # sudo systemctl status keepalived
+   ```
+
+#### Repeat the above steps for second NVA/firewall
+
+#### Setup static route on default Tier-1 gateway
+
+In this section you will set up a static route on default Tier-1 gateway to divert all traffic towards the source NAT'ted public IP towards the NVA/firewalls IP.
+
+1. Login to NSX-T manager.
+2. Click on "Networking".
+3. Click on the three dots just before the default Tier-1 gateway. ![t1-static-route01](../nsx-firewall-patterns/assets/t1-static-route01.jpg)
+4. Click on "Edit" to modify the gateway.
+5. Click on Static Route to expand the section. ![t1-static-route02](../nsx-firewall-patterns/assets/t1-static-route02.jpg)
+6. Click on "Set" to create a new static route.
+7. Add a static route to divert southbound ingress traffic from default Tier-1 towards NVA virtual IP address (North VIP). ![t1-static-route03](../nsx-firewall-patterns/assets/t1-static-route03.jpg)
+
+
+8. Save the configuration.
+
+
+#### Setup static routes on isolated Tier-1 gateway for northbound traffic towards internet
+
+In this section you will create static routes from isolated Tier-1 gateways to firewall/NVA. Due to the limitation of NSX-T, you will be breaking down the default route (0.0.0.0/0) into two sub networks (0.0.0.0/1 and 128.0.0.0/1)
+
+1. Select "Tier-1 Gateways" blade on right. ![t1-static-route04](../nsx-firewall-patterns/assets/t1-static-route04.jpg)
+2. Click three dots on the left to edit the gateway.
+3. Expand "STATIC ROUTES". ![t1-static-route05](../nsx-firewall-patterns/assets/t1-static-route05.jpg)
+4. Click "ADD STATIC ROUTES". ![t1-static-route06](../nsx-firewall-patterns/assets/t1-static-route06.jpg)
+5. Create a default static route.
+   1. Enter "default01" as the name.
+   2. Enter "0.0.0.0/1" as the network.
+   3. Click on "Next Hops" to add one. ![t1-static-route06](../nsx-firewall-patterns/assets/t1-static-route07.jpg)
+6. Set Virtual NVA IP Address as the next hop and set the scope to the segments attached to this Tier-1 gateway. ![t1-static-route07](../nsx-firewall-patterns/assets/t1-static-route08.jpg)
```

Comment:
Is the Scope necessary?

Reply:
@khushal08  Kindly check comments from Kenyon?

Reply:
@khensler Good point. It is optional. Better to remove it to avoid confusion.

Reply:
@khensler @prasad3017  Updated. Any other feedback?